migrator/migrations/m_192_to_m_193_report_config_migrations/migration_impl.go

```diff
@@ -0,0 +1,175 @@
+package m192tom193
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	updatedSchema "github.com/stackrox/rox/migrator/migrations/m_192_to_m_193_report_config_migrations/schema"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/pkg/uuid"
+	"gorm.io/gorm/clause"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+func migrate(database *types.Databases) error {
+	_ = database // TODO(dont-merge): remove this line, it is there to make the compiler happy while the migration code is being written.
+	// Use databases.DBCtx to take advantage of the transaction wrapping present in the migration initiator
+	db := database.GormDB
+	pgutils.CreateTableFromModel(database.DBCtx, db, updatedSchema.CreateTableReportConfigurationsStmt)
+	db = db.WithContext(database.DBCtx).Table("report_configurations")
+
+	dbSnapshot := database.GormDB
+	pgutils.CreateTableFromModel(database.DBCtx, dbSnapshot, updatedSchema.CreateTableReportSnapshotsStmt)
+	dbSnapshot = dbSnapshot.WithContext(database.DBCtx).Table("report_snapshots")
+
+	dbNotifier := database.GormDB
+	pgutils.CreateTableFromModel(database.DBCtx, dbSnapshot, updatedSchema.CreateTableNotifiersStmt)
```

Comment:
`dbNotifier` in second arg
`CreateTableReportConfigurationsStmt.Children[0]` in third arg

Fyi `CreateTableReportConfigurationsStmt.Children[0]` would refer to create statement for